bench/irmin-pack/bench_common.ml

```diff
@@ -93,11 +93,6 @@ let with_progress_bar ~message ~n ~unit =
   in
   with_reporter ~config bar
 
-module Conf = struct
```

Comment:
The CI seems to fail because it misses this module:
```
Error: The implementation bench/irmin-pack/bench_common.pp.ml
       does not match the interface bench/irmin-pack/.bench_common.objs/byte/bench_common.cmi:
        The module `Conf' is required but not provided
       File "bench/irmin-pack/bench_common.mli", line 33, characters 0-31:
         Expected declaration
```
Is it normal to remove it without removing it from the `bench_common.mli`?

Reply:
Nope, just me being sloppy :'-) Looks to be fixed now.

Reply:
It's Friday evening, it happens :wink:  The CI is fixed indeed, thanks!